code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -132,13 +143,17 @@
 										datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
 	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
 	if(!ui)
-		var/datum/asset/assets = get_asset_datum(/datum/asset/simple/pills)
+		var/datum/asset/assets = get_asset_datum(/datum/asset/spritesheet/simple/pills)
 		assets.send(user)
 
 		ui = new(user, src, ui_key, "chem_master", name, 500, 550, master_ui, state)
 		ui.open()
 
-
+//Insert our custom spritesheet css link into the html
+/obj/machinery/chem_master/ui_base_html(html)
```

Comment:
here's the new proc that allows overriding of the base html, we use this to inject our spritesheet html link